mu-plugins/blocks/multisite-latest-posts/src/front.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { createElement, render } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block.js';
+
+const init = () => {
+	const blockElements = document.querySelectorAll( '.wporg-multisite-latest-posts-js' );
+
+	if ( ! blockElements ) {
+		return;
+	}
+
+	for ( let i = 0; i < blockElements.length; i++ ) {
+		const blockEl = blockElements[ i ];
+
+		render( createElement( Block, blockEl.dataset ), blockEl );
```

Comment:
Is there a reason you're using `createElement` rather than JSX here?

Reply:
The habit goes back to the [meeting calendar.](https://github.com/WordPress/meeting-calendar/blob/a16b720dc76a7a4cb27b21a00707af4b301ea645/src/frontend/index.js#L22). I don't mind jsx.